docs/step-by-step/kubeapps-on-tkg/README.md

```diff
@@ -0,0 +1,72 @@
+# Kubeapps on Tanzu Kubernetes Grid
+
+In this guide, we will walk you through performing a production-ready configuration and using Kubeapps on Tanzu Kubernetes Grid (TKG) clusters.
+
+[VMware Tanzu™ Kubernetes Grid™](https://tanzu.vmware.com/kubernetes-grid) is an enterprise-ready Kubernetes runtime that streamlines operations across a multi-cloud infrastructure. It can run both on-premise in vSphere and in the public cloud and includes signed and supported versions of open source applications to provide all the key services required for a production Kubernetes environment.
+
+One of the key aspects of Day 2 operations is managing your workloads. In this scenario, to simplify application deployment and management on Kubernetes, [Kubeapps](https://kubeapps.com/) provides a web-based dashboard to deploy, manage, and upgrade applications on a Kubernetes cluster.
+
+Kubeapps includes a built-in catalog of Helm charts and operators that allows users to deploy trusted and continuously maintained content on their clusters. In addition, you can configure Kubeapps to use your private application repository as its source. This feature gives you the option of extending your catalog with your charts located in a [private Helm repository](https://github.com/kubeapps/kubeapps/blob/master/docs/user/private-app-repository.md) such as ChartMuseum or Harbor, and even to use your customized Helm chart catalog directly from the [VMware Tanzu™ Application Catalog™](https://tanzu.vmware.com/application-catalog).
```

Comment:
built-in catalog -> assuming this is the Bitnami catalog, correct?

Reply:
Yes (I copied this paragraph from somewhere), but I think we should rephrase it. This guide does not assume that kubeapps come with the bitnami catalog by default, afaik